examples/mongocxx/document_validation.cpp

```diff
@@ -0,0 +1,70 @@
+#include <iostream>
+
+#include <bsoncxx/builder/stream/document.hpp>
+#include <bsoncxx/stdx/string_view.hpp>
+
+#include <mongocxx/client.hpp>
+#include <mongocxx/exception/write.hpp>
+#include <mongocxx/instance.hpp>
+#include <mongocxx/stdx.hpp>
+#include <mongocxx/uri.hpp>
+
+using bsoncxx::builder::stream::document;
+using bsoncxx::builder::stream::open_document;
+using bsoncxx::builder::stream::close_document;
+using mongocxx::stdx::string_view;
+using mongocxx::collection;
+using mongocxx::validation_criteria;
+
+int main(int, char**) {
+    mongocxx::instance inst{};
+    // mongocxx::client conn{mongocxx::uri()};
+    mongocxx::client conn{mongocxx::uri{}};
+
+    string_view test("test");
+    auto db = conn[test];
+
+    // Create a collection with document validation enabled.
+    {
+        // @begin: cpp-create-collection-with-document-validation
+        validation_criteria validation;
+        validation.level("strict");
+        validation.action("error");
+
+        // Add a validation rule: all zombies need to eat some brains.
+        document rule;
+        rule << "brains" << open_document << "$gt" << 10 << close_document;
```

Comment:
did you mean `{$gt: 1}` instead of `10`? otherwise Bloody Betty doesn't pass validation.